packages/documentation/src/content/docs/integration/event-handlers.mdx

```diff
@@ -33,7 +33,7 @@ Example: An incoming payment was completed and received $10.
     participant R as Rafiki
 
     R->>ASE: webhook event: incoming payment completed,<br>receivedAmount: $10
-    ASE->>R: admin API call: WithdrawIncomingPaymentLiquidity
+    ASE->>R: admin API call: CreateIncomingPaymentWithdrawal
```

Comment:
should we link somewhere in the document that this would change depending on whether the withdrawal is two-phase or not?

Reply:
(maybe with an `opt` section for this mermaid diagram?)

Reply:
The updated docs.
<img width="661" alt="image" src="https://github.com/interledger/rafiki/assets/1992108/60987965-e6a1-47c2-99f2-7449cc657aa2">

Outgoing payment has also been updated.